spec/features/auth/signup_feature_spec.rb

```diff
@@ -143,6 +143,32 @@
                                   text: 'Password confirmation doesn\'t match Password')
   end
 
+  scenario "Submit Button gets disabled when any field is filled wrong", js: true do
+    visit "/users/sign_up"
+    wait_for_effect_on("#new_user")
+    fill_in "Username", with: user.username
+    fill_in "user_email", with: "gibberish"
+    fill_in "user_password", with: "12341234"
+    fill_in "user_password_confirmation", with: "532"
+    page.execute_script "$('#user_username').trigger('focusout')"
+    wait_for_effect_on("#user_username")
```

Comment:
I did it as @mssola mentioned about test might become flaky
> You can take a look at the spec/features/namespaces_spec.rb file to check some tests using js: true. Moreover, I'd recommend adding a wait method before this: expect(page).to have_button('submit_btn', disabled: true). Otherwise this might become a flaky test I fear :smiley: